config/firestore.rules

```diff
@@ -79,6 +79,12 @@ service cloud.firestore {
         allow write: if isAuthenticated();
     }
 
+    //hackathon dates 
+    match /hackathons/{year} {
+        allow read: if isAuthenticated() && (isAdmin() || isOwner(resource.data.applicantId));
```

Comment:
I'm guessing we want to alloy anyone to read the hackathon dates. Should it be something like this?
```suggestion
        allow read: if isAuthenticated();
```